pyclibrary/c_parser.py

```diff
@@ -1694,24 +1274,26 @@ def _init_cparser(extra_types=None, extra_modifiers=None):
              Word(alphas + "_", alphanums + "_$") +
              WordEnd(wordchars)).setParseAction(lambda t: t[0])
 
-    call_conv = Optional(Keyword('__cdecl') |
-                         Keyword('__stdcall'))('call_conv')
-
     # Removes '__name' from all type specs. may cause trouble.
     underscore_2_ident = (WordStart(wordchars) + ~keyword + '__' +
                           Word(alphanums, alphanums+"_$") +
-                          WordEnd(wordchars)).setParseAction(lambda t: t[0])
-    type_qualifier = ZeroOrMore((underscore_2_ident + Optional(nestedExpr())) |
-                                kwl(qualifiers))
-
-    storage_class_spec = Optional(kwl(storage_classes))
+                          WordEnd(wordchars)
+                          )
 
+    special_quals = underscore_2_ident   ###TODO: remove / has to be done via extra_modifiers in future
```

Comment:
Currently the parser accepts **all** names that begin with '__' as type qualifiers plus some hard coded type qualifiers (i.e. 'const', 'volatile', ...)
But as we plan to provide compiler specific parsers the parser could know exactly the list of type qualifiers to accept.

This means we could replace this generic type qual detector (all words starting with '__' are typequals), by a compiler specific one (all words passed to the Parsers constructur are typequals).